src/transaction-runner.ts

```diff
@@ -26,6 +26,7 @@ import {isSessionNotFoundError} from './session-pool';
 import {Database} from './database';
 import {google} from '../protos/protos';
 import IRequestOptions = google.spanner.v1.IRequestOptions;
+import {protos} from '.';
```

Comment:
`import {google} from '../protos/protos'` is already imported. 

You could import IsolationLevel on top like IRequestOptions

`import IsolationLevel = google.spanner.v1.TransactionOptions.IsolationLevel`

Reply:
done, thanks for the suggestion!